### PR TITLE
handle sent date from sms and mms

### PR DESCRIPTION
**Pull request by lduboeuf**
refs: https://github.com/ubports/messaging-app/issues/179

This allow upper layer to have sent date information

### CODE REVIEW

**Review by Flohack74 — APPROVED** [review]

File: ofonotextchannel.cpp [hunk 1]
```diff
@@ -479,6 +480,7 @@ void oFonoTextChannel::mmsReceived(const QString &id, uint handle, const QVarian
     header["message-token"] = QDBusVariant(id);
     header["message-sender"] = QDBusVariant(handle);
     header["message-received"] = QDBusVariant(QDateTime::currentDateTimeUtc().toTime_t());
+    header["message-sent"] = QDBusVariant(QDateTime::fromString(info["SentTime"].toString(), Qt::ISODate).toTime_t());
```
Comment (Flohack74) [inline ofonotextchannel.cpp]: You need to define the variable info here, otherwise:
`error: 'info' was not declared in this scope`

**Review by Flohack74 — CHANGES_REQUESTED** [review]
Compile fixes needed

File: ofonotextchannel.cpp (hunk 1 above)
Comment (Flohack74) [reply ofonotextchannel.cpp]: Basically that means that info must be filled first from the data like it is done for the SMS

Comment (lduboeuf) [reply ofonotextchannel.cpp]: oups sorry

**Comment by lduboeuf** [comment]
some choices need to be done regarding managing the sent date, see ubports/messaging-app#179

**Comment by lduboeuf** [comment]
please double check for mms as it is not reliable on my device. e.g i can only receive a mms if i have data enabled. If flight mode during someone send a mms, i will not receive it later

**Pull request merged**